data/mainnet/pools.js

```diff
@@ -4,6 +4,204 @@ const addresses = require('./addresses.json')
 const strat30PercentFactor = '0.7'
 
 module.exports = [
+  {
+    chain: CHAINS_ID.MATIC_MAINNET,
+    id: 'quickGamma_MATIC_ETH_narrow',
+    type: POOL_TYPES.INCENTIVE,
+    tradingApyFunction: {
+      type: TRADING_APY_TYPES.QUICKSWAP_GAMMA,
+      params: [addresses.MATIC.V2.quickGamma_MATIC_ETH_narrow.Underlying],
+    },
+    contractAddress: addresses.MATIC.V2.quickGamma_MATIC_ETH_narrow.NewPool,
+    collateralAddress: addresses.MATIC.V2.quickGamma_MATIC_ETH_narrow.NewVault,
+    rewardAPY: [],
+    rewardAPR: null,
+    rewardTokens: [addresses.MATIC.miFARM],
+    rewardTokenSymbols: ['miFARM', 'WMATIC', 'dQUICK'],
+    stakeAndDepositHelpMessage: `
+    <div class="help-message">
+      <b>Deposit and stake:</b>
+      <ol class="numeric-list">
+        <li>
+          Go to&nbsp;
+          <a target="_blank" rel="noopener noreferrer" href="https://quickswap.exchange/#/pools?currency0=${addresses.MATIC.WMATIC}&currency1=${addresses.MATIC.pWETH}">
```

Comment:
Wrong link, we need a link to quickswap.gamma.xyz for the correct pool. Same for the other pools